misc/scripts/add-repo.sh

```diff
@@ -49,7 +53,7 @@ else
 	fi
 fi
 
-if ! wget -q --spider -- "$REPO/packagelist"; then
+if ! curl --head -s --fail -- "$REPO/packagelist" > /dev/null; then
```

Comment:
```suggestion
if ! curl --head --location -s --fail -- "$REPO/packagelist" > /dev/null; then
```

Makes it follow redirects.